source/leadership.rst

```diff
@@ -0,0 +1,133 @@
+encompasses many members of the community, but hopefully will help to direct
+and distribute project member energy.
+
+**Red Group** (includes executive committee, and all subproject leaders) –
+Members of this group spend a large part of their time organizing the
+yt-project from a high level. They are expected to coordinate their actions
+with other red members so that the project presents a coherent position towards
+the outside. They are expected to prepare and attend the monthly team calls and
+executive committee meetings. We expect red members to teach and grow other
+members so that they can pass their organizational responsibilities on to them.
+This encourages us to increase diversity, reduce burn-out and allow group
+members to shift their focus in reaction to changing life circumstances. If a
+red member chooses to change focus, that change of focus will be announced at
+the next monthly team meeting. Red group members are expected to also perform
+duties of blue group members while facilitating and helping the blue group to
+grow.
+
+**Blue Group** (includes all subproject deputies and yt project members) – This
+is where the journey of a new yt project team member starts. Blue group members
+spend a large part of your time facilitating contributions from others to the
+project (e.g. reviewing work, answering questions, maintaining project
+infrastructure, doing triage of new issues). We expect blue members to teach
+and grow contributors that can become blue members. This encourages diversity
+of the project. After building a good understanding of the technology and
+social structure of the project blue members can change to a red member by
+volunteering for tasks which a red member is expected to perform. After a while
+of doing so the group will ask if you want to become a red member and, if you
+agree, recognise you as such at a monthly team meeting.
+
+**Green Group** (can be any person, any time, self-nominated only)-- This group
+applies to members of the community whose life situation has changed so that
+they prefer to (temporarily) not take on the rights and responsibilities of
+being an active member. Membership to the green group is invoked by
+self-nomination. They can return to active membership by resuming their
+community activities. Their return will be announced at the next team meeting
+or via the developers mailing list.
+
+**Emeritus Group** (time based or self-nominated)-- this group is for members
+of the community at any level that have a prolonged leave of member duties from
+the yt project. We welcome participation in the community for any length of
+time, and this is a way to retire from the project. A project member may become
+emeritus by self-nomination or by not interacting in a membership capacity on
+the yt project for over 2 years. After this time, a member of the executive
+committee will contact an inactive member and see if they wish to remain active
+or transition to an emeritus role. 
```

Comment:
Assuming this goes in, we should probably identify who this would include.

Reply:
Another thing we might want to discuss is that these guidelines do not require transitioning to emeritus if somebody is inactive for over two years. They're asked if they want to transition, but they could also request to be a part of the "green group" if they don't want to become emeritus. I feel weird the idea of forcing emeritusship on members even if they're inactive.

Reply:
I think we should remove commit access to the repo from people who haven't participated in more than some time period. This is mostly because of the security implications. Forcing everyone to have 2FA on their github accounts would ameliorate this concern.

Reply:
I agree, we should transition the group, and I think 2 years is a good amount.  We should probably make sure we're measuring all types of interaction, not just the types that need a commit bit.